internal/audit/cerboshub/conf.go

```diff
@@ -30,54 +32,82 @@ var (
 )
 
 type Conf struct {
-	local.Conf
 	Ingest IngestConf `yaml:"ingest"`
+	// Local is configuration for the local Audit backend. See 'audit.local' section for documentation
+	Local local.Conf `yaml:"local" conf:",example=see audit.local section"`
```

Comment:
Could we embed this struct or name it something other than `local`? I think it would be confusing for some users if we redirect them to the `local` config. Ideally, everything they need to configure the `hub` backend should be visible in a single documentation block. 

If `confdocs` is unable to handle embedded structs, we should fix that. @oguzhand95  can probably take a look at that in parallel.

Reply:
Perhaps embedding would be the tidiest option. Seems like the existing config parsing logic doesn't take embedded structs into account, I can take a look at that.

Separately, when I tinkered with this a couple of weeks ago, I came across issues with `confdocs` generation as well. Seems like it gets a bit fiddly because in it's existing form, it only has access to the local struct. I think we'd need to somehow cache traversed nodes or pass state around so that `audit.(cerbos)hub` can access the struct metadata in `audit.local`. I [stubbed out the part where it hits embedded nodes](https://github.com/cerbos/cerbos/blob/f63a63d2a0194d731b29ae5dfdff416a7b5e1868/hack/tools/confdocs/confdocs.go#L243-L245) in the generation logic as a temp measure. Help appreciated @oguzhand95!

Reply:
I've fixed the yaml inlining so can move onto the `confdocs` generation now (in case you read this Oguzhan).